### DISCUSSION TITLE
Are there any known issues with 10.6.9 and BRAW toolbox?

### DISCUSSION THREAD

**Discussion by jefforig** [latenitefilms/BRAWToolbox#155] [Support]
We are having some issues on a machine where we had to manually update from 10.6.3 to 10.6.6 to be compatible with a different library from a different machine.

This machine that we manually updated is having serious issues and I am considering upgrading to 10.6.9

Are there any known issues with 10.6.9 and BRAW toolbox?

Thank you!

**Comment by latenitefilms** [comment]
No, there are no known issues or changes in the latest Final Cut Pro 10.6.9 release.
